### PR TITLE
style: Remove commented code from rikaikun.

### PR DESCRIPTION
**Pull request by melink14**
In some cases, code comments were replaced with clearer prose comments.

---
This change is [Reviewable](https://reviewable.io/reviews/melink14/rikaikun/269)

### CODE REVIEW

File: extension/data.ts [hunk 1]
```diff
@@ -99,7 +96,7 @@ RcxDict.prototype = {
 
   fileReadArray: function (name, charset) {
     const a = this.fileRead(name, charset).split('\n');
-    // Is this just in case there is blank shit in the file.  It was writtin
+    // Is this just in case there is blank shit in the file.  It was written
```
Comment (Stephie) [inline extension/data.ts:99]: 😂

File: extension/data.ts [hunk 2]
```diff
@@ -146,13 +143,9 @@ RcxDict.prototype = {
     );
   },
 
-  //  Note: These are mostly flat text files; loaded as one continous string to
+  //  Note: These are mostly flat text files; loaded as one continuos string to
```
Comment (Stephie) [inline extension/data.ts:146]:
```suggestion
  //  Note: These are mostly flat text files; loaded as one continuous string to
```

File: extension/rikaichan.ts [hunk 3]
```diff
@@ -333,14 +168,11 @@ window.rcxMain = {
         .catch(function (err) {
           alert('Error loading dictionary: ' + err);
         });
-
-      //  time = (((new Date()).getTime() - time) / 1000).toFixed(2);
     }
   },
 
-  // This function diables
+  // This function disables
```
Comment (Stephie) [inline extension/rikaichan.ts:174]: Finish this comment

File: extension/rikaicontent.ts [hunk 4]
```diff
@@ -435,10 +403,9 @@ const rcxContent = {
         let ofs = ev.currentTarget.rikaichan.uofs;
         for (i = 50; i > 0; --i) {
           ev.currentTarget.rikaichan.uofs = --ofs;
-          // chrome.extension.sendMessage({"type":"resetDict"});
-          // this.showMode = 0;
           if (this.show(ev.currentTarget.rikaichan, this.defaultDict) >= 0) {
-            if (ofs >= ev.currentTarget.rikaichan.uofs) break; // ! change later
```
Comment (Stephie) [inline extension/rikaicontent.ts:441]: Keep this ancient comment for now

**Review by Stephie — APPROVED** [review]
🗑️ 👍

File: extension/data.ts (hunk 1 above)
Comment (melink14) [inline extension/data.ts:99]: What can you do about that shit?!?

File: extension/data.ts (hunk 2 above)
Comment (melink14) [reply extension/data.ts:146]: Done.

File: extension/rikaichan.ts (hunk 3 above)
Comment (melink14) [reply extension/rikaichan.ts:174]: done.

File: extension/rikaicontent.ts (hunk 4 above)
Comment (melink14) [reply extension/rikaicontent.ts:441]: done.

**Pull request merged**